cmake/scripts/common/Macros.cmake

```diff
@@ -662,7 +662,11 @@ function(core_find_git_rev stamp)
       string(REPLACE "\"" "" DATE ${DATE})
       string(REPLACE "-" "" DATE ${DATE})
     else()
-      string(TIMESTAMP DATE "%Y%m%d" UTC)
+      if(EXISTS ${CMAKE_SOURCE_DIR}/BUILDDATE)
+        file(STRINGS ${CMAKE_SOURCE_DIR}/BUILDDATE DATE LIMIT_INPUT 16)
```

Comment:
`LIMIT_INPUT 8` is enough for YYYYmmdd

Reply:
That does make a lot of sense. WIll force push in a sec.